packages/platform-sdk/test/integration-tests/cart/cart.test.ts

```diff
@@ -52,7 +55,22 @@ import {
 } from '../product/product-fixture'
 
 describe('testing cart API calls', () => {
-  //create
+  let category,
+    taxCategory,
+    productType,
+    product1,
+    product2,
+    cart1,
+    cart2,
+    cart3,
+    cart4,
+    cart5,
```

Comment:
is there a possibility to change the name like cartWithCustomer or cartWithTwoProducts so having some meaningful name of variables?
You can check also in java how these variables are called.